test-it/ManualLoaderTests/loaderglobal_currentbrowserfps.js

```diff
@@ -11,9 +11,10 @@
 
     var runTest = function () {
         var viaCode = document.getElementById('viacode').textContent;
-        var vireo = new window.NationalInstruments.Vireo.Vireo();
-        vireo.eggShell.loadVia(viaCode);
-        vireo.eggShell.executeSlicesUntilClumpsFinished(function () {
+        window.vireoHelpers.createInstance().then(function (vireo) {
```

Comment:
Can we use the "await" keyword here to avoid having to string "then"s together?

Reply:
done, definitely cleaner